### PR TITLE
fix(Makefile): failing missing tiller directory

### PR DESCRIPTION
**Pull request by naveensrinivasan**
The clean command fails when `/rootfs/tiller` is missing.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/helm/1360)

### CODE REVIEW

**Comment by technosophos** [comment]
Thanks! I think Makefile.orig accidentally got merged in here during a rebase. Can you remove it?

File: Makefile [hunk 1]
```diff
@@ -86,6 +86,7 @@ protoc:
 .PHONY: clean
 clean:
 	@rm -rf $(BINDIR) ./rootfs/tiller ./_dist
+	@rm -rf ./rootfs/tiller
```
Comment (adamreese) [inline Makefile]: The tiller directory is being removed in the line above

**Comment by naveensrinivasan** [comment]
I had an old version of the makefile that was causing this to fail. Figured out it isn't an issue. So closing the PR.

**Comment by technosophos** [comment]
Thanks for the PR nonetheless!

**Pull request closed without merging**